llvm/include/llvm/TargetParser/Triple.h

```diff
@@ -46,64 +46,64 @@ class Triple {
   enum ArchType {
     UnknownArch,
 
-    arm,            // ARM (little endian): arm, armv.*, xscale
```

Comment:
Please do not do such large code reformatting. It makes impossible to review them

Reply:
without doing code formatting the CI won't pass.

Reply:
You can ignore these.

Reply:
i will fix it later. but first pass CI with other things first to ensure it does not break anything.